reporter/html.go

```diff
@@ -135,15 +135,39 @@ func (h *htmlReporter) Build() ([]byte, error) {
 			return "info"
 		},
 	}
+	buildNonPathChangelogJson, err := json.Marshal(h.buildNonPathChangelogList())
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+
+	buildPathChangelogJson, err := json.Marshal(h.buildPathChangelogMap())
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+
+	buildStatusJson, err := json.Marshal(h.output.ExecutionStatus)
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+
 
 	data := struct {
-		Status               differentiator.ExecutionStatus
-		NonPathChangelogList []differentiator.Changelog
-		PathChangelogList    []pathKeyValue
+		Status                   differentiator.ExecutionStatus
+		StatusJson				 string
+		NonPathChangelogList     []differentiator.Changelog
+		PathChangelogList        []pathKeyValue
+		NonPathChangelogListJson string
+		PathChangelogListJson 	 string
 	}{
-		Status:               h.output.ExecutionStatus,
-		NonPathChangelogList: h.buildNonPathChangelogList(),
-		PathChangelogList:    h.buildPathChangelogMap(),
+		Status:                   h.output.ExecutionStatus,
```

Comment:
according to "ui/public/index.html" we need just "Status" & "PathChangelogList" (in json format) - we should clean everything else